src/rcx_tk/process_metadata_file.py

```diff
@@ -32,9 +33,9 @@ def process_metadata_file(file_path):
     df = read_file(file_path)
     df = df[list(columns_to_keep.keys())].rename(columns=columns_to_keep)
     df['sampleName'] = df['sampleName'].str.replace(' ', '_')
-    save_dataframe_as_tsv(df, file_path)
+    save_dataframe_as_tsv(df, out_path)
 
-def process_alkane_ri_file(file_path):
+def process_alkane_ri_file(file_path, out_path):
     """Processes an Alkane RI file, keeping and renaming specific columns."""
```

Comment:
Add type annotations and include proper docstring which also explains the parameters.

Reply:
resolved by [65ef8c9](https://github.com/RECETOX/rcx-tk/pull/13/commits/65ef8c9f11b624803534989304c88f3a596de3f9)